src/SamplesApp/UITests.Shared/Windows_Devices/Midi/MidiDeviceInput.xaml.cs

```diff
@@ -0,0 +1,257 @@
+					break;
+				case MidiMessageType.ControlChange:
+					var controlChangeMessage = (MidiControlChangeMessage)receivedMidiMessage;
+					outputMessage.Append(", Channel: ").Append(controlChangeMessage.Channel).Append(", Controller: ").Append(controlChangeMessage.Controller).Append(", Value: ").Append(controlChangeMessage.ControlValue);
+					break;
+				case MidiMessageType.ProgramChange:
+					var programChangeMessage = (MidiProgramChangeMessage)receivedMidiMessage;
+					outputMessage.Append(", Channel: ").Append(programChangeMessage.Channel).Append(", Program: ").Append(programChangeMessage.Program);
+					break;
+				case MidiMessageType.ChannelPressure:
+					var channelPressureMessage = (MidiChannelPressureMessage)receivedMidiMessage;
+					outputMessage.Append(", Channel: ").Append(channelPressureMessage.Channel).Append(", Pressure: ").Append(channelPressureMessage.Pressure);
+					break;
+				case MidiMessageType.PitchBendChange:
+					var pitchBendChangeMessage = (MidiPitchBendChangeMessage)receivedMidiMessage;
+					outputMessage.Append(", Channel: ").Append(pitchBendChangeMessage.Channel).Append(", Bend: ").Append(pitchBendChangeMessage.Bend);
+					break;
+				case MidiMessageType.SystemExclusive:
+					var systemExclusiveMessage = (MidiSystemExclusiveMessage)receivedMidiMessage;
+					outputMessage.Append(", ");
+
+					// Read the SysEx bufffer
+					var sysExDataReader = DataReader.FromBuffer(systemExclusiveMessage.RawData);
+					while (sysExDataReader.UnconsumedBufferLength > 0)
+					{
+						byte byteRead = sysExDataReader.ReadByte();
+						// Pad with leading zero if necessary
+						outputMessage.Append(byteRead.ToString("X2")).Append(" ");
+					}
+					break;
+				case MidiMessageType.MidiTimeCode:
+					var timeCodeMessage = (MidiTimeCodeMessage)receivedMidiMessage;
+					outputMessage.Append(", FrameType: ").Append(timeCodeMessage.FrameType).Append(", Values: ").Append(timeCodeMessage.Values);
+					break;
+				case MidiMessageType.SongPositionPointer:
+					var songPositionPointerMessage = (MidiSongPositionPointerMessage)receivedMidiMessage;
+					outputMessage.Append(", Beats: ").Append(songPositionPointerMessage.Beats);
+					break;
+				case MidiMessageType.SongSelect:
+					var songSelectMessage = (MidiSongSelectMessage)receivedMidiMessage;
+					outputMessage.Append(", Song: ").Append(songSelectMessage.Song);
+					break;
+				case MidiMessageType.TuneRequest:
+					var tuneRequestMessage = (MidiTuneRequestMessage)receivedMidiMessage;
+					break;
+				case MidiMessageType.TimingClock:
+					var timingClockMessage = (MidiTimingClockMessage)receivedMidiMessage;
+					break;
+				case MidiMessageType.Start:
+					var startMessage = (MidiStartMessage)receivedMidiMessage;
+					break;
+				case MidiMessageType.Continue:
+					var continueMessage = (MidiContinueMessage)receivedMidiMessage;
+					break;
+				case MidiMessageType.Stop:
+					var stopMessage = (MidiStopMessage)receivedMidiMessage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'stopMessage'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)